samples/kvsWebRTCClientMaster.c

```diff
@@ -120,6 +127,9 @@ INT32 main(INT32 argc, CHAR *argv[])
             THREAD_JOIN(pSampleConfiguration->audioSenderTid, NULL);
         }
 
+        if(pSampleConfiguration->enableFileLogging) {
+            RELEASE_FILE_LOGGER();
```

Comment:
I would prefer the equivalent freeFileLogger API instead of the macro. Not a strong feeling though. I was just thinking that those would be used in pairs like CREATE_FILE_LOGGER and RELEASE_FILE_LOGGER but we are using the API directly when creating/setting

Reply:
Done